source/ProcessManager.Core/Infrastructure/Diagnostics/HealthChecks/OrchestrationRegisterHealthCheck.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Energinet.DataHub.ProcessManager.Core.Infrastructure.Registration;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace Energinet.DataHub.ProcessManager.Core.Infrastructure.Diagnostics.HealthChecks;
+
+/// <summary>
+/// A health check to verify the health of the orchestration register. Returns <see cref="HealthStatus.Unhealthy"/> if
```

Comment:
Not sure how we can write this, but we don't know for sure its "HealthStatus.Unhealthy"

```suggestion
/// A health check to verify the health of the orchestration register. Returns "FailureStatus" if
```